src/sagemaker/pytorch/estimator.py

```diff
@@ -215,10 +233,28 @@ def __init__(
 
         self.distribution = distribution or {}
 
+    def _pytorch_distribution_configuration(self, distribution):
+        """Returns a dict of distribution config for PyTorch training
+
+        Args:
+            distribution (dict): A dictionary with information on how to run distributed training.
+        Returns:
+            dict containing Pytorch DDP config
+        """
+        distribution_config = {}
+        if "pytorchddp" in distribution:
+            pytorch_ddp_enabled = distribution.get("pytorchddp").get("enabled", False)
```

Comment:
What if the user set up `pytorchddp` as `False`, and set normal distribution config. looks these lines didn't work for this case.

Reply:
Makes sense, fixed.